lighthouse-cli/test/smokehouse/smokehouse-report.js

```diff
@@ -0,0 +1,224 @@
+/* eslint-disable no-console */
```

Comment:
Only changes here are 1) module exports at the bottom and 2) using `Smokehouse.` type definitions.